examples/twittermap/noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/FileFeedDriver.java

```diff
@@ -6,13 +6,14 @@
 import org.kohsuke.args4j.Option;
 
 import java.io.*;
+import java.util.Random;
 
 public class FileFeedDriver {
 
     @Option(required = true,
             name = "-u",
             aliases = "--url",
-            usage = "url of the feed adapter")
+            usage = "url or IP(s) of the feed adapter")
```

Comment:
Just a thought:
Is it better if we add a few words saying "use , to separate multiple urls or IPs." ?